flowzone.yml

```diff
@@ -1617,12 +1620,20 @@ jobs:
           node-version: 18
 
       - name: Docusaurus Builder
-        uses: product-os/docusaurus-builder@master
+        if: |
+          needs.project_types.outputs.has_readme == 'true' &&
+          inputs.cloudflare_website != '' && inputs.custom_website_build != 'true'
+        uses: product-os/docusaurus-theme@master
```

Comment:
We should pin this to a version. Renovate will update it as needed.

Reply:
Yeah, sounds good.